### DISCUSSION TITLE
Export deliverables in Romanian

### DISCUSSION THREAD

**Discussion by senny7** [monarc-project/MonarcAppFO#417] [Q&A]
Hello,

Is it possible to export deliverables in Romanian? I just started using Monarc (awesome project 👍 ) and I'm running a local MonarcFO deployment v 2.11.1. I noticed that in the GUI under Home > Administration > Deliverable Templates there are Romanian options on all template categories. However,  when I try to add one, the web app only lists French, English, German and Dutch .docx upload options. 

I have also noticed that the raw templates are stored under ./deliveries/cases/. I am willing to translate the templates, if necessary, and ideally I would like to know if there is an option to make the templates available for export within the GUI. I expect that other objects such as reference sets (standards) require separate translations and import via MOSP. 

Thanks!

**Comment by ruslanbaidan** [comment]
Hello,

Romanian language was added to Monarc user interface recently, thanks to a contribution by a person from Romania who made it.
Deliverables/reports can be translated to Romanian and imported instead of the language that is used for analysis creation.
However, as you mentioned, the objects will be in the language of your analysis.
There is no convenient way of translating the library objects to any other language for now besides of default ones.
We are going to work on it in the future to make it possible.

Thank you.

Regards,
